source/includes/table-beta-compatibility.rst

```diff
@@ -50,11 +56,29 @@
      - |checkmark|
      - 
      - |checkmark|
+     - |checkmark|
 
    * - Destination Data Handling 
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
-     - 
\ No newline at end of file
+     - 
+     - |checkmark|
```

Comment:
BTW I noticed that the staging links include mongosync 1.9 beta instead of 1.8. Can we ensure that the beta version is 1.8 still?

Reply:
Our current master branch is set to the `upcoming` version which is 1.9. This change will be backported to 1.8 so the page that users are directed to will be updated, too.

Reply:
sg, thanks!